taxi/views.py

```diff
@@ -110,9 +155,7 @@
 @login_required
 def toggle_assign_to_car(request, pk):
     driver = Driver.objects.get(id=request.user.id)
-    if (
-        Car.objects.get(id=pk) in driver.cars.all()
-    ):  # probably could check if car exists
+    if Car.objects.get(id=pk) in driver.cars.all():
         driver.cars.remove(pk)
     else:
         driver.cars.add(pk)
```

Comment:
In the `toggle_assign_to_car` function, consider handling the case where the car or driver might not exist, which could raise an exception. Using `get_object_or_404` might be more appropriate here.